### ISSUE TITLE
Testimonial Page - Heading elements are not in a sequentially-descending order

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#220]
**Describe the bug**
Heading elements are not in a sequentially-descending order on the Testimonial . See [here](https://web.dev/heading-order/?utm_source=lighthouse&utm_medium=devtools)

** Failing Element** 
`<h3>Help Us Teach More Veterans How To Code &nbsp;</h3>`

**To Reproduce**
1. Go to vetswhocode.io
2. Run lighthouse report
3. View accessibility report.

**Expected behavior**
The element should be ordered correctly based on the current markup. 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
